js/models/addressBook_model.js

```diff
@@ -13,9 +13,56 @@ app.factory('AddressBook', function()
 					}
 				}
 				return undefined;
+			},
+
+			sharedWith: {
+				users: [],
+				groups: []
 			}
 
 		});
 		angular.extend(this, data);
+		angular.extend(this, {
+			owner: data.url.split('/').slice(-3, -2)[0]
```

Comment:
I have to double check this ...

Actually I wanted to ask you what for the owner is used.
Best would be to get it from the profind response ...